R/transform-files.R

```diff
@@ -81,18 +81,39 @@ transform_file <- function(path,
 make_transformer <- function(transformers) {
   force(transformers)
   function(text) {
-    transformed_text <- parse_transform_serialize(text, transformers)
-    transformed_text
+    transformed_code <- text %>%
+      parse_transform_serialize_r(transformers) %>%
+      parse_transform_serialize_roxygen(transformers)
+    transformed_code
   }
 }
 
+#' @importFrom purrr map_at flatten_chr
+parse_transform_serialize_roxygen <- function(text, transformers) {
+  roxygen_seqs <- identify_start_to_stop_of_roxygen_examples_from_text(text)
```

Comment:
Probably need to split that up into multiple functions. Also, some of it might be re-used for dontrun handling.